### PR TITLE
Get netplugin working

### PR DESCRIPTION
**Pull request by erikh**
This gets netplugin working again on volplugin's setup.

### CODE REVIEW

**Comment by mapuri** [comment]
LGTM

**Comment by erikh** [comment]
note this branch is still waiting on a patch for netplugin that has a bug that the volplugin fd leak test exposes.

**Comment by erikh** [comment]
Ok, merging this.

**Comment by erikh** [comment]
Actually no, need to fix up some commits first. I will merge after the demo.

**Comment by erikh** [comment]
I am confident this branch is ready now. Please re-review if you have time.

**Comment by erikh** [comment]
@mapuri @unclejack PTAL

File: Vagrantfile [hunk 1]
```diff
@@ -52,6 +52,7 @@ ansible_provision = proc do |ansible|
   # In a production deployment, these should be secret
   ansible.extra_vars = {
     docker_version: "1.10.2",
+    swarm_bootstrap_node_name: "mon0",
```
Comment (mapuri) [inline Vagrantfile]: just a heads up, this variable is no longer needed in the latest ansible.

**Comment by mapuri** [comment]
@erikh is this PR mostly vendoring in new ansible and Vagrantfile changes? I think we can vendor new ansible but upto you, otherwise LGTM.

File: Vagrantfile (hunk 1 above)
Comment (erikh) [inline Vagrantfile]: Oh ok, I'll refactor.

**Comment by erikh** [comment]
Yeah, as of yesterday's ansible.

Most of the patch contains fixups to the ansible to enable netplugin in 
volplugin, then the latest is merged on top of it.

**Pull request merged**